pontoon/base/models.py

```diff
@@ -2756,5 +2783,17 @@ def calculate_stats(self, save=True):
         if project_locale:
             project_locale.adjust_stats(
                 total_strings_diff, approved_strings_diff,
-                fuzzy_strings_diff, unreviewed_strings_diff
+                fuzzy_strings_diff, unreviewed_strings_diff,
+                for_dummy_project=resource.project.is_dummy
             )
+
+        # Locale
+        total_strings_diff = (total_strings_diff, 0)[resource.project.is_dummy]
+        approved_strings_diff = (approved_strings_diff, 0)[resource.project.is_dummy]
+        fuzzy_strings_diff = (fuzzy_strings_diff, 0)[resource.project.is_dummy]
+        unreviewed_strings_diff = (unreviewed_strings_diff, 0)[resource.project.is_dummy]
+        locale.adjust_stats(
+            total_strings_diff, approved_strings_diff,
+            fuzzy_strings_diff, unreviewed_strings_diff,
+            for_dummy_project=False
+        )
```

Comment:
What's the point of this repeated code? Why not just use the same pattern as for other calls to `adjust_stats`?

Reply:
Actually, these are  Locales used for all the projects(normal and dummy ones). Other objects are related to specifically dummy  project which can be easily set to zero but doing the same breaks the stat calculation for other projects. If I m still unable to make it clear enough then we can discuss it once.